### PR TITLE
VerityMocks and VerifyAllMocks Static Helper Methods

### PR DESCRIPTION
**Pull request by RehanSaeed**
I tend to write a method that does multiple verifications a lot, it saves a few lines:

```
mock1.VerifyAll();
mock2.VerifyAll();
mock3.VerifyAll();

// Versus

Mock.VerifyAll(mock1, mock2, mock3);
```

### CODE REVIEW

File: Source/Mock.cs [hunk 1]
```diff
@@ -121,6 +121,24 @@ protected Mock()
 
 			throw new ArgumentException(Resources.ObjectInstanceNotMock, "mocked");
 		}
+		
+		/// <include file='Mock.xdoc' path='docs/doc[@for="Mock.VerifyMocks"]/*'/>
+		public static void VerifyMocks(params Mock[] mocks)
```
Comment (kzu) [inline Source/Mock.cs]: I think I like the original proposal naming better: `Verify(mocks)`

File: Source/Mock.cs [hunk 2]
```diff
@@ -121,6 +121,24 @@ protected Mock()
...
+        	{
+            		foreach (var mock in mocks)
+            		{
+                		mock.Verify();
+            		}
+        	}
+		
+		/// <include file='Mock.xdoc' path='docs/doc[@for="Mock.VerifyAllMocks"]/*'/>
+		public static void VerifyAllMocks(params Mock[] mocks)
```
Comment (kzu) [inline Source/Mock.cs]: Ditto here, I like `VerifyAll(mocks)` better

**Comment by kzu** [comment]
If these two get renamed to Verify and VerifyAll, I'll merge it ;)

**Comment by RehanSaeed** [comment]
@kzu Methods renamed.

**Pull request merged**